ovnnotify.go

```diff
@@ -33,8 +33,11 @@ func (notify ovnNotifier) Stolen([]interface{}) {
 }
 func (notify ovnNotifier) Echo([]interface{}) {
 }
+
 func (notify ovnNotifier) Disconnected(client *libovsdb.OvsdbClient) {
```

Comment:
dont need verify client == self?

Reply:
It should be ok. libovsdb maintains the connections table for each connection and its CBs, and only calls the CB for the handler of the disconnected client.